src/games/strategy/triplea/ui/MapPanel.java

```diff
@@ -140,6 +143,16 @@ public void mouseReleased(final MouseEvent e) {
         if (terr != null) {
           notifyTerritorySelected(terr, md);
         }
+        if(e.getButton() == 4 || e.getButton() == 5){
+          if (is4Pressed && is5Pressed) {
+            lastActive = e.getButton() == 4 ? 5 : 4;
+          } else {
+            lastActive = -1;
```

Comment:
extract all magic numbers to constants please, allows them to be referenced by name instead of numbers.